### PR TITLE
Initial repo cleanup

### PR DESCRIPTION
**Pull request by dominicaw**
### 🚧 Changed
We are moving to Github releases instead of manual file versioning. See this [Slab doc](https://offerzen.slab.com/posts/webflow-script-cdn-etijpmeo) for more information.

With this in mind, we have deleted all old versions of scripts and only kept the most recent ones.
Some script names have to be kept as they are, for example `navigation@6.2.js` as it is currently being used on Webflow and will break if we delete it. We will do another cleanup after this PR has been merged and Webflow script embeds have been updated.

Some scripts we are not totally sure where they are being used, so they have been kept in place, for example `community-activities@2.0`

### CODE REVIEW

File: offerzen/global/employers/employers.css
Comment (jethroflanagan) [inline offerzen/global/employers/employers.css:1]: Is the @1.0 version of this in use?

File: global/country-dropdown.css
Comment (jethroflanagan) [inline global/country-dropdown.css:1]: Is 1.0 in use?

File: global/country-dropdown.js
Comment (jethroflanagan) [inline global/country-dropdown.js:1]: Is 1.0 in use?

**Review by jethroflanagan — APPROVED** [review]
I think it seems fine as I don't know what's actually live - but I added some questions to just check you still retain copies of the "live" versions e.g. employers@1.0.css (if that is in use)

File: global/country-dropdown.js
Comment (dominicaw) [reply global/country-dropdown.js:1]: nope! couldnt find this in use in Webflow but just left a copy in case

File: global/country-dropdown.css
Comment (dominicaw) [reply global/country-dropdown.css:1]: Also not in use on Webflow

File: offerzen/global/employers/employers.css
Comment (dominicaw) [reply offerzen/global/employers/employers.css:1]: Nope, couldnt find it in use on Webflow :)

**Review by jethroflanagan — APPROVED** [review]

**Pull request merged**